### DISCUSSION TITLE
Error TS5023 while running scully

### DISCUSSION THREAD

**Discussion by seiuro2** [scullyio/scully#1392] [General]
✗ npm run scully --pluginsError=false 

> wearemo-portal@0.0.0 scully
> scully

using plugins from folder "./scully"
Typescript error while compiling plugins. the error is:
error TS5023: Unknown compiler option '-'.

⠹ undefined

**Comment by seiuro2** [comment]
hello! i need some help here i dont have any idea  of this problem, but  the last tipe i clone the project and tray to run this thing appears

**Reply by Jefiozie** [reply]
Hi @seiuro2 would it be possible to share a reproduction repo? This way we can have a better look on your problem.

**Comment by seiuro2** [comment]
Hi! thanks for your answer, i dont have permission to do that,  but i
identify the problem its local because some of my teammates can run the
project without problem, but i dont know how to solve it and work with.
i check my version of typescript, angular, vs code and we think we have the
same but still without work... if you have any suggestions im all ears and
eyes!

El jue, 17 jun 2021 a las 11:08, Jeffrey Bosch ***@***.***>)
escribió:

> Hi @seiuro2 <https://github.com/seiuro2> would it be possible to share a
> reproduction repo? This way we can have a better look on your problem.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scullyio/scully/discussions/1392#discussioncomment-884186>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AP4W6AIR35PCHU5HC3FQKMLTTIMYZANCNFSM4635DTIA>
> .
>


-- 
*José García.*
Desarrollador Web FrontEnd
Email: ***@***.***
GitHub: seiuro2
+57 350 543 40 84

**Reply by SanderElias** [reply]
can you remove `node_mudules` then `npm install` and try again? This sounds like an environmental issue.
If we can't reproduce it, there isn't that much we can do to help.
